default.nix

```diff
@@ -0,0 +1,83 @@
+let
+  inputs = import ./flake-inputs.nix;
+in
+  {
+    nixpkgs ? inputs.nixpkgs,
+    dream2nix ? inputs.dream2nix,
+    rust-overlay ? inputs.rust-overlay,
+    sops-nix ? inputs.sops-nix,
+    self ? {
+      dirtyRev = "dirty";
+      lastModifiedDate = "19700101";
+    },
```

Comment:
What exactly do we need that for?

Reply:
Overview, which inherits `inputs` from `flake.nix` to get info about the current repo (which I guess is invalid without Flakes).

Reply:
Resolved by forking flake-compat...